### PR TITLE
Upgrade to Laravel 8

### PR DESCRIPTION
**Pull request by daniel-g-wood**
Updates dependencies to Laravel 8, including moving to the new version of Model Factories.

### CODE REVIEW

File: database/factories/ResultFactory.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Studio\Totem\Result;
+
+class ResultFactory extends Factory
```
Comment (qschmick) [inline database/factories/ResultFactory.php]: ❤️

**Comment by qschmick** [comment]
@daniel-g-wood Looks great, really appreciate the factory refactoring 😄 Can you fix the few small styling issues?

**Comment by daniel-g-wood** [comment]
@qschmick Sorry about that! My first pull request, and I wasn't exactly sure what to do... Should be right now.

**Comment by roshangautam** [comment]
Thank you @daniel-g-wood Merged and released v8.0

**Pull request merged**